test/E2E_Test/E2ETestBase.cs

```diff
@@ -12,24 +13,26 @@ namespace Microsoft.VisualStudio.Web.CodeGeneration.E2E_Test
 {
     public class E2ETestBase
```

Comment:
Yes, this is the bulk of it. 
The renames of txt files are just to use them as resources of the assembly.